test/built-ins/Temporal/PlainMonthDay/from/fields-missing-properties.js

```diff
@@ -12,4 +13,7 @@ assert.throws(TypeError, () => Temporal.PlainMonthDay.from({ day: 15 }), "Only d
 assert.throws(TypeError, () => Temporal.PlainMonthDay.from({ monthCode: 'M12' }), "Only monthCode");
 assert.throws(TypeError, () => Temporal.PlainMonthDay.from({ monthCode: undefined, day: 15 }), "monthCode undefined");
 assert.throws(TypeError, () => Temporal.PlainMonthDay.from({ months: 12, day: 31 }), "months plural");
-assert.throws(TypeError, () => Temporal.PlainMonthDay.from({ month: 11, day: 18, calendar: "iso8601" }), "month, day with calendar");
+
+if (TemporalHelpers.nonDefaultCalendarId) {
+  assert.throws(TypeError, () => Temporal.PlainMonthDay.from({ month: 11, day: 18, calendar: TemporalHelpers.nonDefaultCalendarId }), "month, day with non-iso8601 calendar");
+}
```

Comment:
The expected failure is now dependent upon use of a non–ISO-8601 calendar.

Reply:
I'm not sure that this is a change we want to make. See https://github.com/tc39/proposal-temporal/pull/2663#issuecomment-1688877251

Reply:
Case 3.ii